app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -24,6 +24,11 @@ class MusicsListAdapter(private val musics: ArrayList<Music>) :
     override fun onBindViewHolder(holder: BaseViewHolder<Music>, position: Int) {
         holder.onBind(musics[position])
     }
+
+    fun submitList(musicList: List<Music>) {
+        musics += musicList
```

Comment:
in list nabayad toye constructor gerefte beshe
baye ye parameter khode class bashe
vali in taghyir e dorostie va khobe Afarin

Reply:
`notifyDataSetChanged` kheili function sanginie va tosie mishe azash dori beshe
sade tarin rah estefade az `ListAdapter<T, V>` hast bejaye class Adapter mamoli